### PR TITLE
Propagate changes from fork to parent repo

### PR DESCRIPTION
**Pull request by ekumachidi**

### CODE REVIEW

**Comment by marc1uk** [comment]
* Can you add in the file `html-Detector/stylesheets/material.min.js.map` from WCTE branch, this fixes some console errors. I would copy it into `html-Common` and add symlinks in `html-Standalone` and `html-Detector`.
* Please remove the emacs backup file `html-Detector/#login.html#`
* I would probably keep the capitalized of L in `html-StandAlone/Logs`.
* We can probably move the `stylesheets` directories to `html-Common`
* We can probably migrate the Config pages to `html-Common` as they're duplicates?
* We can probably remove `html-Detector/Testing`
* We can probably remove html-Detector/Config/test`
* One for @brichards64 but do we need both SubSystemTemplate and SubSystemExample?
* I think the `monitoringplots` folder needed to be in an `html` directory to be accessible (at least with our current apache configuration), so I would keep it there. This would need to update `/cgi-bin/monitoringplots.cgi` as well. I'm not sure if @brichards64 wants to add symlinks in `html-Detector` and/or `html-Standalone`.
* Please remove (or comment out) `ServiceSend.setsockopt(ZMQ_IMMEDIATE, b);` in `cgi-bin/sendcommand2nopadding.cpp` as that turned out to be a mistake. It causes the socket to hang if there's no listener, which is the opposite of what we want.
* maybe also copy over the extra debugging line from `cgi-bin/sqlquery.cgi` from wcte. Small change but nice to have.
* I believe this isn't used and is probably going to be removed, but the changes in `html-Common/Monitoring/plots.js` are needed to prevent errors.
* I would keep the wcte changes in `html-Common/SQL/sql.js`.
* The changes in `html-Common/includes/functions.js` need to be migrated.
* I think the last 3 lines added in `html-Detector/includes/drawer.html` are superfluous and cause issues.


* This is probably moot as it'll be replaced with evgenii's changes, but we should make sure we keep track of things like exporting `PGHOST` and `PGUSER` in scripts such as `cgi-bin/get_event.cgi` and `cgi-bin/pmt-locations.cgi`. Your version hard-codes localhost, the wcte version checks for those variables and overrides with a WCTE-specific IP address. Neither is really great, and we shouldn't lose track of these.

Think that's all the changes I can spot.

**Comment by brichards64** [comment]
alarms folder should be a sym link to alarms in common. Please check changes are propagated to common

**Comment by brichards64** [comment]
remove testing folder

**Comment by brichards64** [comment]
please change to tooldaq logo favicon

**Comment by brichards64** [comment]
remove wcte specific subsystem pages

File: cgi-bin/sqlquery.cgi [hunk 1]
```diff
@@ -11,19 +11,22 @@ user=${post[user]}
 db=${post[db]}
 command=${post[command]}
 # TODO make an argument?
-PGHOST=127.0.0.1
+PGHOST=192.168.10.17
```
Comment (brichards64) [inline cgi-bin/sqlquery.cgi:14]: should be loop back

File: html-Common/Data/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+sure go for it
```
Comment (brichards64) [inline html-Common/Data/README.md:1]: do we need?

**Comment by ekumachidi** [comment]
> remove wcte specific subsystem pages

Already addressed

**Comment by ekumachidi** [comment]
> remove wcte specific subsystem pages

Done

**Comment by brichards64** [comment]
style sheets folder needs sym linking not individual files. logs folder in standalone is lowercase L

**Comment by brichards64** [comment]
We can probably migrate the Config pages to html-Common as they're duplicates?

**Pull request merged**